GitUI/Translation/English.xlf

```diff
@@ -586,6 +586,10 @@ Please make sure git (msysgit or cygwin) is installed or set the correct command
         <source>Reset Unstaged Changes</source>
         <target />
       </trans-unit>
+      <trans-unit id="chkAddNewlineToCommitMessageWhenMissing.Text">
+        <source>Add newline after first line in commit message when missing</source>
```

Comment:
It is more about an empty line that is missing. Maybe: Ensure the second line of commit message is empty.